YunoHost/yunohost
# enh: Handle dyndns subscribe/unsubscribe in web admin

## Solution

Adapt a bit the subscription for the web admin.  
Most of the changes while adding a domain is about trying to unsubscribe the domain if it is not available and a recovery password is set, before trying to subscribe to it again.

## PR Status

Needs moar tests.
We may have to change dynette's requests limiter value. One is allowed to subscribe/unsubscribe 5 times per hour only.

## How to test

- add a dyndns domain with a recovery password
- check that a password is set in the domain's config panel
- delete the domain without subscribing to it
- try to readd the domain with the recovery password

- add a dyndns domain without recovery password
- set one with the config panel
- delete the domain without subscribing to it
- try to readd the domain with the recovery password
